components/courses/app/models/courses/interview.rb

```diff
@@ -22,9 +22,8 @@ def mentor_name
       mentor.full_name
     end
 
-    # need to be changed after creating a Student model
     def student_name
```

Comment:
you don't need this method, just:

`delegate :full_name, to: :student, allow_nil: true`